docs/modules/ROOT/pages/tutorials/getting-started.adoc

```diff
@@ -1,483 +1,184 @@
-+
-[source,shell]
-----
-kubectl cluster-info
-----
---
-
-[TIP]
-.Getting started on macOS with Docker Desktop for Mac
---
-. Use https://docs.docker.com/docker-for-mac/[Docker Desktop for Mac] and enable https://docs.docker.com/docker-for-mac/#kubernetes[Kubernetes]
-. Install the https://kubernetes.github.io/ingress-nginx/deploy/#docker-for-mac[Nginx ingress]. This won't work if you have other services running on port 80 and 443 on your macOS.
---
+Project Syn provides the tools to help you manage many Kubernetes clusters from a central location:
 
-== Kickstart Lieutenant
+. Lieutenant keeps the inventory of tenants and clusters, using a set of Git repositories as the storage mechanism;
+. Steward acts as the cluster agent, wrapping https://argoproj.github.io/cd/[Argo CD], who keeps an eye on said Git repositories;
+. And Commodore serves as the tool used to manipulate the cluster records, saving changes to the Git repositories whenever needed.
 
-Lieutenant is the central inventory service all Project Syn managed clusters report to, it consists of the Lieutenant _Operator_ and the Lieutenant _API_.
+In this guide, we're going to focus on Lieutenant and Steward. The xref:tutorials/getting-started-2.adoc[second part] will focus on Commodore.
 
-As you can see on the xref:about/architecture.adoc[architecture] diagram, Lieutenant is the central API which is being used by all other Project Syn tools. Therefore it needs to be installed first so that all other components are actually able to do their job.
-
-Create a namespace to host the Operator and the API
-[source,shell]
-----
-kubectl create namespace lieutenant
-----
-
-=== Install Lieutenant Operator
-
-Install *Lieutenant Operator* with the following commands:
-
-[source,shell,subs="attributes"]
-----
-# CRDs (global scope)
-kubectl apply -k "github.com/projectsyn/lieutenant-operator/config/crd?ref={lieutenant_operator_version}"
-
-# Operator deployment
-kubectl -n lieutenant apply -k "github.com/projectsyn/lieutenant-operator/config/samples/deployment?ref={lieutenant_operator_version}"
-
-# Operator configuration
-kubectl -n lieutenant set env deployment/lieutenant-operator -c lieutenant-operator \
-  DEFAULT_DELETION_POLICY=Delete \
-  DEFAULT_GLOBAL_GIT_REPO_URL=https://github.com/projectsyn/getting-started-commodore-defaults \
-  LIEUTENANT_DELETE_PROTECTION=false \
-  SKIP_VAULT_SETUP=true
-----
-
-These environment variables will configure Lieutenant Operator to:
+== Requirements
 
-* not set deletion protection annotations (`DEFAULT_DELETION_POLICY`)
-* configure the global git repo URL by default on Tenants (`DEFAULT_GLOBAL_GIT_REPO_URL`)
-* delete external resources by default (good for cleaning up after this guide is finished) (`LIEUTENANT_DELETE_PROTECTION`)
-* not use Hashicorp Vault (don't do this for production deployments) (`SKIP_VAULT_SETUP`)
+Before you start, please check the following requirements:
 
-=== Install Lieutenant API
+* A GitLab account.
```

Comment:
Any specific versions required?

Reply:
Not that I'm aware of. It works with GitLab.com, which is GitLab 15 afaik.

Reply:
I was referring to the whole list, not only GitLab. It was just a thought. If versions are not important, leave it as is.